.github/workflows/metabox.yaml

```diff
@@ -50,9 +50,17 @@ jobs:
       - name: Checkout Checkbox monorepo
         uses: actions/checkout@v3
       - name: Setup LXD
-        uses: canonical/setup-lxd@90d76101915da56a42a562ba766b1a77019242fd
-      - name: Install pyopenssl
-        run: python3 -m pip install pyopenssl --upgrade
+        uses: canonical/setup-lxd@v0.1.1
+      - name: Add ZFS storage
+        run: |
```

Comment:
Please the very least comment the rationale for doing this, and the mechanism by which this is crucial for speed. Does the storage pool naming match something in a pre-existing implementation for example? Should the implementation _expect_ that ZFS is used?